src/test/java/seedu/address/logic/parser/FetchBillCommandParserTest.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+//import static seedu.address.logic.parser.CliSyntax.PREFIX_ID;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ROOMNUMBER;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+//import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.FetchBillCommand;
+import seedu.address.model.hotel.Room;
+import seedu.address.model.hotel.person.Person;
+import seedu.address.model.ids.PersonId;
+import seedu.address.testutil.PersonBuilder;
+import seedu.address.testutil.RoomBuilder;
+
+public class FetchBillCommandParserTest {
+    private static final Person Guest = new PersonBuilder().build();
+    private static final PersonId PersonId = Guest.getPersonId();
+    private static final Room Room = new RoomBuilder().build();
+    private static final String RoomNum = Room.getRoomNum();
+
+    private FetchBillCommandParser parser = new FetchBillCommandParser();
+    /*
+    @Test
```

Comment:
Why are these commented out?

Reply:
They cause checkstyle errors due to personbuilder, thus as mentioned above I need to figure this out. ReserveCommandParserTest seems to have the same issue too.